docs/configure/nodes.md

```diff
@@ -61,6 +61,10 @@ The Netdata config directory also contains one symlink:
   versions are copied into the config directory when opened with `edit-config`. _Do not edit the files in
   `/usr/lib/netdata/conf.d`, as they are overwritten by updates to the Netdata Agent._
 
+## Configure a Netdata docker container
+
+See [configure agent containers](https://learn.netdata.cloud/docs/agent/packaging/docker#configure-agent-containers).
```

Comment:
```suggestion
See [configure agent containers](packaging/docker/README.md#configure-agent-containers).
```

Perhaps can be converted to an internal link. Not sure if that is better cc @tkatsoulas